uftrace.c

```diff
@@ -153,6 +154,7 @@ static struct argp_option uftrace_options[] = {
 	{ "kernel-only", OPT_kernel_only, 0, 0, "Dump kernel data only" },
 	{ "flame-graph", OPT_flame_graph, 0, 0, "Dump recorded data in FlameGraph format" },
 	{ "sample-time", OPT_sample_time, "TIME", 0, "Show flame graph with this sampling time" },
+	{ "dot", OPT_dot, 0, 0, "Dump recorded data in DOT format" },
```

Comment:
I think the name "dot" is too short and vague.  I prefer "graphviz" instead - what do you think?